src/toil/test/__init__.py

```diff
@@ -98,10 +99,15 @@ def tearDown(self):
     @classmethod
     def awsRegion(cls):
         """
-        Use us-west-2 unless running on EC2, in which case use the region in which
-        the instance is located
+        If TOIL_AWS_ZONE is set, use that.
+        Else use "us-west-2" unless running on EC2, in which case
+        use the region in which the instance is located
         """
-        return cls._region() if running_on_ec2() else 'us-west-2'
+        return zone_to_region(
+            os.environ.get(
+                "TOIL_AWS_ZONE", cls._region() if running_on_ec2() else "us-west-2"
```

Comment:
This isn't nested right; we're passing some things that are already regions to `zone_to_region()`.

Reply:
I'll remove this once there is a targeted test for just `us-east-1`.